test/qml/qt5/tst_map.qml

```diff
@@ -1,12 +1,13 @@
-// Copyright (C) 2023 MapLibre contributors

```

Comment:
We only have a single copyright year: https://github.com/maplibre/maplibre-gl-js/blob/main/LICENSE.txt#L1

Reply:
Habit from work 😊

Fixed.